compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -2212,6 +2212,8 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
       extension (self: TypeLambda)
         def param(idx: Int): TypeRepr = self.newParamRef(idx)
         def paramBounds: List[TypeBounds] = self.paramInfos
+        def paramVariances: List[Flags] =
```

Comment:
Having this just on `TypeLambda` is a bit limiting since classes also have parameter with variance annotations, it would be nice if the same API worked for TypeLambda, TypeRef of classes and TypeRef of parameterized type aliases.

Reply:
I added `Symbol.paramVariance` and added tests for all these cases. I did not add it on `TypeRef` because it is more consistent if we go to the definition of the class and ask for the variances on it.